k8s/server/overlays/ephemeral/django/deployment.yaml

```diff
@@ -7,7 +7,10 @@ spec:
     spec:
       containers:
       - name: server
-        image: northamerica-northeast1-docker.pkg.dev/pht-01hp04dtnkf/hopic-k8s-images/cpho-phase2:prod-207fc8eb-1711642657 # {"$imagepolicy": "flux-system:server"}
+        image: northamerica-northeast1-docker.pkg.dev/pht-01hp04dtnkf/hopic-k8s-images/cpho-phase2:distroless-runtime-and-management-image-aec37750-1712599872 # {"$imagepolicy": "flux-system:distroless-server"}
+        securityContext:
```

Comment:
I noticed that the base deployment has a `securityContext` block, do we need to explicitly define it here as well?